Source/ACE/Entity/WorldObject.cs

```diff
@@ -1963,7 +1991,7 @@ private WeenieHeaderFlag SetWeenieHeaderFlag()
             if (AmmoType != null)
                 weenieHeaderFlag |= WeenieHeaderFlag.AmmoType;
 
-            if (Value != null)
+            if (Value != null && (Value > 0))
```

Comment:
The example of this would be doors. Many doors have a value of 0 but they are not sent with weenieflags.

You can have a value of 0, it is never sent as part of the weenie flag, only upon ID